src/components/FormPluginEditor/TipTapEditor.styles.scss

```
@@ -0,0 +1,162 @@
+            blockquote {
+                border-left: 5px solid #eee;
+                color: #666;
+                font-style: italic;
+                margin: 16px 0;
+                padding: 10px 20px;
+            }
+
+            pre {
+                background-color: rgba(0, 0, 0, 0.05);
+                border-radius: 4px;
+                font-family: 'Inconsolata', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inconsolata" (font-family-name-quotes)

(font-family-name-quotes)